routers/api/v1/repo/release_tags.go

```diff
@@ -55,6 +55,11 @@ func GetReleaseTag(ctx *context.APIContext) {
 		return
 	}
 
+	if err != nil && models.IsErrReleaseNotExist(err) || release.IsTag {
```

Comment:
this is already checked!!! (at L48)

the only new thing is the `release.IsTag` check ... witch will break things

If we are going this way, we have to invent a new endpoint for git tags only